pymeasure/instruments/lakeshore/lakeshore3xx.py

```diff
@@ -0,0 +1,91 @@
+    The LS 336  has four input channels (A, B, C and D) and four output channels (1, 2, 3 and 4).
+
+    The LS 340 has two input channels (A and B) and one output channel (1).
+    Following extension cards can be used:
+    * 3465 Single Capacitance Input Option Card adds one channel (C).
+    * 3462 Dual Standard Input Option Card adds two channels (C and D).
+    * 3464 Dual Thermocouple Input Option Card adds two channels (C and D).
+    * 3468 Eight Channel Input Option Card adds eight channels (C1-C4, D1-D4).
+
+    The LS 350 has four input channels (A, B, C and D) and four output channels (1, 2, 3 and 4).
+    Following extension card can be used:
+    * 3062 4-Channel Scanner option adds 4 additional channels (D2, D3, D4, and D5).
+
+    This driver makes use of the :ref:`LakeShoreChannels`.
+
+    .. code-block:: python
+
+        controller = LakeShore3xx("GPIB::1")
+
+        print(controller.output_1.setpoint)         # Print the current setpoint for loop 1
+        controller.output_1.setpoint = 50           # Change the loop 1 setpoint to 50 K
+        controller.output_1.heater_range = 'low'    # Change the heater range to low.
+        controller.input_A.wait_for_temperature()   # Wait for the temperature to stabilize.
+        print(controller.input_A.temperature)       # Print the temperature at sensor A.
+    """
+    input_A = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'A')
+    input_B = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'B')
+    input_C = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'C')
+    input_D = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'D')
+    input_C1 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'C1')
+    input_C2 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'C2')
+    input_C3 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'C3')
+    input_C4 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'C4')
+    input_D1 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'D1')
+    input_D2 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'D2')
+    input_D3 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'D3')
+    input_D4 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'D4')
+    input_D5 = Instrument.ChannelCreator(LakeShoreTemperatureChannel, 'D5')
+    output_1 = Instrument.ChannelCreator(LakeShoreHeaterChannel, 1)
+    output_2 = Instrument.ChannelCreator(LakeShoreHeaterChannel, 2)
+    output_3 = Instrument.ChannelCreator(LakeShoreHeaterChannel, 3)
+    output_4 = Instrument.ChannelCreator(LakeShoreHeaterChannel, 4)
+
+    def __init__(self,
+                 adapter,
+                 name="Lakeshore Model 3xx Temperature Controller",
+                 read_termination="\r\n",
+                 **kwargs):
+        kwargs.setdefault('read_termination', read_termination)
+        super().__init__(
+            adapter,
+            name,
+            **kwargs
```

Comment:
As the kwargs are set to the super call, the following is better:


```suggestion
        super().__init__(
            adapter,
            name,
            read_termination=read_termination,
            **kwargs
```